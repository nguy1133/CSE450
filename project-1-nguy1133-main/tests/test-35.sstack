// ERROR: Cannot PUSH from an illegal variable
PUSH "Value"
POP my_var
PUSH not_a_var
