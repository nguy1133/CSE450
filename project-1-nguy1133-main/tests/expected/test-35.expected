ERROR (line 4): Unknown variable 'not_a_var'
